generated/nidaqmx/system/device.py

```diff
@@ -1161,6 +1174,27 @@ def unreserve_network_device(self):
 
     # endregion
 
+    def get_ext_cal_last_date_and_time(device_name):
```

Comment:
Instance methods need a `self` parameter.
```suggestion
    def get_ext_cal_last_date_and_time(self):
```

Reply:
Missed out the `self` parameter, it's added.